src/components/apps/dungeon/home.tsx

```diff
@@ -1718,12 +1732,12 @@ export function DungeonApp() {
                                 imageRendering: "pixelated",
                             }}
                             src={power_potion}
-                            width={EMOJI_SIZE}
+                            width={POTION_SIZE}
                             alt={""}
                         />
                     </Box>
-                    <Text pt={EMOJI_SIZE / 2} className="font-face-sfpb" color="white" fontSize="10px">
-                        {current_player_data === null ? "" : "x" + current_player_data?.num_advantage_potions}
+                    <Text pt={POTION_SIZE/2} className="font-face-sfpb" color="white" fontSize={isMobile ? "10px" :"14px"}>
```

Comment:
does prettier not enforce POTION_SIZE / 2?

Reply:
Not sure what you mean by enforce .